packages/router/src/AuthenticatedRoute.tsx

```diff
@@ -3,10 +3,19 @@ import React, { useCallback } from 'react'
 import { Redirect } from './links'
 import { routes } from './router'
 import { useRouterState } from './router-context'
+import type { GeneratedRoutesMap } from './util'
 
-export function AuthenticatedRoute(props: any) {
+interface AuthenticatedRouteProps {
```

Comment:
Pulled this change out into its own PR if we want to keep this one cleaner https://github.com/redwoodjs/redwood/pull/8989